Description: More rpath tweak removal
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-08-05

Index: python-pyeclib/setup.py
===================================================================
--- python-pyeclib.orig/setup.py	2015-08-05 08:40:04.549702848 +0000
+++ python-pyeclib/setup.py	2015-08-05 08:43:55.692013250 +0000
@@ -95,10 +95,6 @@
     for prefix in ('/', '/usr', '/usr/local', _exec_prefix):
         libdir = os.path.join(prefix, 'lib')
         libdir64 = os.path.join(prefix, 'lib64')
-        if arch64 and os.path.exists(libdir64):
-            default_library_paths.append(libdir64)
-        else:
-            default_library_paths.append(libdir)
     return default_library_paths
 
 
@@ -223,7 +219,6 @@
 
         installroot = install_lib.install_dir
 
-        default_library_paths.insert(0, installroot)
         _install.run(self)
 
         # Another Mac-ism...  If the libraries are installed
