Description: More rpath tweak removal
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-11-14

Index: python-pyeclib/setup.py
===================================================================
--- python-pyeclib.orig/setup.py
+++ python-pyeclib/setup.py
@@ -46,9 +46,7 @@ from setuptools.command.install import i
 platform_str = platform.platform()
 default_python_incdir = get_python_inc()
 default_python_libdir = get_python_lib()
-default_library_paths = [default_python_libdir,
-                         ('%s/usr/local/lib' % _exec_prefix),
-                         '/lib', '/usr/lib', '/usr/local/lib']
+default_library_paths = [default_python_libdir]
 
 # utility routine
 def _read_file_as_str(name):
@@ -130,7 +128,6 @@ class install(_install):
         if installroot.startswith("/usr"):
             installroot = "/"
 
-        default_library_paths.insert(0, "%s/usr/local/lib" % installroot)
         _install.run(self)
 
         # Another Mac-ism...  If the libraries are installed
@@ -163,11 +160,8 @@ module = Extension('pyeclib_c',
                    define_macros=[('MAJOR VERSION', '1'),
                                   ('MINOR VERSION', '0')],
                    include_dirs=[default_python_incdir,
-                                 '/usr/local/include',
-                                 '/usr/local/include/jerasure',
                                  '/usr/include',
-                                 'src/c/pyeclib_c',
-                                 '/usr/local/include'],
+                                 'src/c/pyeclib_c'],
                    library_dirs=default_library_paths,
                    runtime_library_dirs=default_library_paths,
                    libraries=['erasurecode'],
